.github/workflows/build.yml

```diff
@@ -89,7 +89,7 @@ jobs:
           tar zxvf ${{ steps.branch_env.outputs.fullname }}
 
       - name: Linux Get dependencies
-        run: sudo apt install -y cpanminus build-essential libncurses5-dev libreadline-dev libssl-dev perl
+        run: sudo apt install -y cpanminus build-essential libncurses5-dev libreadline-dev libssl-dev perl libpcre3 libpcre3-dev
```

Comment:
Hi @rushitote I think you need to also install pcre lib in chaos testing ci to make it pass, see [error log](https://github.com/apache/apisix/pull/4710/checks?check_run_id=3207414954#step:4:932). 
You could add it before building docker image [here](https://github.com/apache/apisix/blob/master/.github/workflows/chaos.yml#L32)

Reply:
@Yiyiyimu Thanks for this, I have made the change.

Reply:
Sorry I made a mistake 😿 We need to add the dependency to the dockerfile but not on the host machine. Do you mind file a PR about the change on the dockerfile: https://github.com/apache/apisix-docker/blob/master/alpine-local/Dockerfile, which is used as building apisix image used in chaos testing

Also we need to change docs about dependencies installation to inform users to install them: https://github.com/apache/apisix/blob/master/docs/en/latest/install-dependencies.md

Reply:
@Yiyiyimu Thanks again, I have created a PR for that at [apisix-docker#202](https://github.com/apache/apisix-docker/pull/202) and added PCRE in the dependency installation doc.